marklogic-data-hub/src/main/java/com/marklogic/hub/impl/HubProjectImpl.java

```diff
@@ -478,7 +478,9 @@ private void writeResourceFile(String srcFile, Path dstFile) {
 
     private void writeResourceFile(String srcFile, Path dstFile, boolean overwrite) {
         if (overwrite || !dstFile.toFile().exists()) {
-            logger.info("Getting file: " + srcFile);
+            if (logger.isDebugEnabled()) {
+                logger.debug("Getting file: " + srcFile);
```

Comment:
I'm guessing we are using the guard condition in this case since we are performing string concatenation.
No need to change right now but the following approach would allow us to get rid of the "if" block.
Since we are using slf4j we can get rid of the guard condition by using 
```suggestion
                logger.debug("Getting file: {}",  srcFile);
```
This gets rid of the superfluous string concatenation when the logger is disabled for DEBUG level.

Reply:
There are a lot of little things like that to clean up - I'm just changing the logging from info to debug here, since the info logging was cluttering up logging in a test. And this doesn't seem like it warrants "info" level.